Shared/Samples/Set visibility of subtype sublayer/SetVisibilityOfSubtypeSublayerView.Model.swift

```diff
@@ -0,0 +1,164 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SwiftUI
+import ArcGIS
+
+extension SetVisibilityOfSubtypeSublayerView {
+    /// The view model for this sample.
+    class Model: ObservableObject {
+        /// A map with a streets basemap style.
+        let map = Map(basemapStyle: .arcGISStreetsNight)
+        
+        /// A feature table for the electrical network in this sample.
+        private let featureTable = ServiceFeatureTable(url: .featureServiceURL)
+        
+        /// A subtype feature layer created from the service feature table.
+        private var subtypeFeatureLayer: SubtypeFeatureLayer!
```

Comment:
Are there guidelines on when to use implicitly unwrapped optionals in the samples app code? I would rather the 3 iuo's here be optionals since they don't get their values in the init. However, if it's common practice to use iuo's like this then that's okay.